src/current/cockroachcloud/folders.md

```diff
@@ -27,6 +27,11 @@ To move a folder or a cluster, you update its `parent_id`. If you move a folder
 - A violation of the [folder structure](#folder-structure) limitations.
 - An attempt to move a folder into itself or into one of its descendant folders.
 
+### Folder naming
+
+- Folder names must begin and end with a letter or number. The only allowed special characters are spaces, hyphens, and underscores.
+- Folder names cannot exceed 40 characters.
+
```

Comment:
Looks good! Can you please include that folder names must be at least 3 characters.